client/landing/gutenboarding/stores/onboard/reducer.ts

```diff
@@ -168,6 +155,19 @@ const selectedFeatures: Reducer< FeatureId[], OnboardAction > = (
 	return state;
 };
 
+const selectedFonts: Reducer< FontPair | undefined, OnboardAction > = (
```

Comment:
These were alphabetized in https://github.com/Automattic/wp-calypso/pull/45145 — let's keep them tidy. :-) 

Also nicer to review this PR without unrelated changes.